backend/prisma/schema.prisma

```diff
@@ -102,6 +137,22 @@ enum InviteStatus {
   NOT_APPLICABLE
 }
 
+model GiftChoice {
```

Comment:
were these gift tables required in this pr? i think this was pulled over from hyun's pr right? if it's not directly relevant to your work let's rm

Reply:
Okay I will remove anything thats not related to user and its contacts